### PR TITLE
Improve formatting for ses_identity_notification_topic doc

### PR DESCRIPTION
**Pull request by multani**
This pull request improves slightly the rendering of [the `aws_ses_identity_notification_topic` resource's documentation](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ses_identity_notification_topic).

### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

### CODE REVIEW

File: website/docs/r/ses_identity_notification_topic.markdown [hunk 1]
```diff
@@ -25,18 +25,18 @@ resource "aws_ses_identity_notification_topic" "test" {
 
 The following arguments are supported:
 
-* `topic_arn` - (Optional) The Amazon Resource Name (ARN) of the Amazon SNS topic. Can be set to "" (an empty string) to disable publishing.
-* `notification_type` - (Required) The type of notifications that will be published to the specified Amazon SNS topic. Valid Values: *Bounce*, *Complaint* or *Delivery*.
+* `topic_arn` - (Optional) The Amazon Resource Name (ARN) of the Amazon SNS topic. Can be set to ``""`` (an empty string) to disable publishing.
```
Comment (justinretzolk) [inline website/docs/r/ses_identity_notification_topic.markdown:28]:
```suggestion
* `topic_arn` - (Optional) The Amazon Resource Name (ARN) of the Amazon SNS topic. Can be set to `""` (an empty string) to disable publishing.
```

File: website/docs/r/ses_identity_notification_topic.markdown [hunk 2]
```diff
@@ -25,18 +25,18 @@ resource "aws_ses_identity_notification_topic" "test" {
...
+* `notification_type` - (Required) The type of notifications that will be published to the specified Amazon SNS topic. Valid Values: ``Bounce``, ``Complaint`` or ``Delivery``.
```
Comment (justinretzolk) [inline website/docs/r/ses_identity_notification_topic.markdown:29]:
```suggestion
* `notification_type` - (Required) The type of notifications that will be published to the specified Amazon SNS topic. Valid Values: `Bounce`, `Complaint` or `Delivery`.
```

File: website/docs/r/ses_identity_notification_topic.markdown [hunk 3]
```diff
@@ -25,18 +25,18 @@ resource "aws_ses_identity_notification_topic" "test" {
...
 * `identity` - (Required) The identity for which the Amazon SNS topic will be set. You can specify an identity by using its name or by using its Amazon Resource Name (ARN).
-* `include_original_headers` - (Optional) Whether SES should include original email headers in SNS notifications of this type. *false* by default.
+* `include_original_headers` - (Optional) Whether SES should include original email headers in SNS notifications of this type. ``false`` by default.
```
Comment (justinretzolk) [inline website/docs/r/ses_identity_notification_topic.markdown:31]:
```suggestion
* `include_original_headers` - (Optional) Whether SES should include original email headers in SNS notifications of this type. `false` by default.
```

**Review by justinretzolk — CHANGES_REQUESTED** [review]
Thank you for your contribution! This looks good to me overall. Suggested changes are to move from double backticks to single backticks to match [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code). I think double backticks technically _does_ work, but single backticks falls in line with what Is done elsewhere in the documentation.

**Comment by multani** [comment]
> Suggested changes are to move from double backticks to single backticks to match [markdown syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#quoting-code). I think double backticks technically _does_ work, but single backticks falls in line with what Is done elsewhere in the documentation.

:facepalm: you are totally right, this is fixed now :+1: (I spent too much time with [reStructuredText](https://docutils.sourceforge.io/docs/user/rst/quickref.html#inline-markup) in the past I think...)

**Review by justinretzolk — APPROVED** [review]
Looks good to me now! Thank you again for the contribution! Once the checks finish up, I'll get this merged in 🚀

**Pull request merged**